spec/Subscriber-spec.ts

```diff
@@ -97,4 +97,15 @@ describe('Subscriber', () => {
     expect(sub1.closed).to.be.false;
     expect(sub2.closed).to.be.true;
   });
+
+  it('should call complete observer without any arguments', () => {
+    const observer = {
+      complete: sinon.spy()
+    };
+
+    const sub1 = new Subscriber(observer);
+    sub1.complete();
+
+    expect(observer.complete.firstCall.args).to.empty;
```

Comment:
yeah, probably simple test like this could be achieved in those way. I'll update test.